classes/DepositObjectDAO.inc.php

```diff
@@ -84,14 +84,10 @@ public function markHavingUpdatedContent($journalId, $objectType) {
 				foreach ($result as $row) {
 					$depositObject = $this->getById($journalId, $row->deposit_object_id);
 					$deposit = $depositDao->getById($depositObject->getDepositId());
-					if($deposit->getSentStatus() || !$deposit->getTransferredStatus()) {
```

Comment:
It's ok to always send a new deposit, the staging server and the LOCKSS software are ready for that.